src/main/resources/db/migration/V1__create_user_table.sql

```diff
@@ -0,0 +1,14 @@
+
+CREATE TABLE APPUSER
+(
+    id serial,
+    first_name varchar(255),
+    last_name  varchar(255),
+    phoneNumber  varchar(255),
```

Comment:
255 for phone number ?

Reply:
consequences of solving to many issues at the time :)